swagger.yml

```diff
@@ -24,15 +24,15 @@ paths:
           description: "TweetQA API successfully read"
           schema:
             type: "string"
-  /controllers/v1/QAModelController/read_qa_model/{uuid}:
+  /v1/model/read_qa_model/{id}:
```

Comment:
I would prefer a format more like /v1/models/{id}

We don't really need to specify the action we are going to take in the url.  The resource we are performing on coupled by the request type (GET/POST/PUT) should be enough.  In general, a format of /version/resource is what you want for creation, /version/resource/id, for updating and retrieving specifics.  That pattern makes things much easier to digest as a client as well.